core/src/components/tab-button/tab-button.ionic.scss

```diff
@@ -86,3 +90,75 @@
 :host(.tab-button-shape-rectangular) {
   @include globals.border-radius(globals.$ion-border-radius-0);
 }
+
+// Tab Button: Badge
+// --------------------------------------------------
+
+:host ::slotted(ion-badge.badge-xxsmall),
+:host ::slotted(ion-badge.badge-xsmall),
+:host ::slotted(ion-badge.badge-small),
+:host ::slotted(ion-badge.badge-medium),
+:host ::slotted(ion-badge.badge-large),
+:host ::slotted(ion-badge.badge-xlarge) {
+  min-width: ion-vars.$tab-ionic-button-badge-size;
+  height: ion-vars.$tab-ionic-button-badge-size;
+
+  font-size: ion-vars.$tab-ionic-button-badge-font-size;
+
+  line-height: ion-vars.$tab-ionic-button-badge-line-height;
+}
```

Comment:
This isn't needed. I was able to remove it on my end and verified that the sizes are still correct since we are using the correct size="xxsmall" prop.

```suggestion
```

Reply:
Same as before.
Done!